src/Raven.NewClient/Commands/StreamOperation.cs

```diff
@@ -68,6 +131,17 @@ private static void ReadNextToken(Stream stream, UnmanagedJsonParser parser, Jso
             }
         }
 
+        private static async void ReadNextTokenAsync(Stream stream, UnmanagedJsonParser parser, JsonOperationContext.ManagedPinnedBuffer buffer)
```

Comment:
`async void` methods are to be avoided. They don't do what you thing you should be doing.